uploadNeuroDB/NeuroDB/objectBroker/ConfigOB.pm

```diff
@@ -93,6 +93,9 @@ use constant BIDS_DATASET_AUTHORS        => 'bids_dataset_authors';
 use constant BIDS_ACKNOWLEDGMENTS_TEXT   => 'bids_acknowledgments_text';
 use constant BIDS_README_TEXT            => 'bids_readme_text';
 use constant BIDS_VALIDATOR_OPTIONS_TO_IGNORE => 'bids_validator_options_to_ignore';
+use constant CREATE_CANDIDATES           => 'createCandidates';
+use constant CREATE_VISIT                => 'createVisit';
+use constant DEFAULT_COHORT              => 'default_cohort';
```

Comment:
missing the following here:
```suggestion
use constant DEFAULT_COHORT              => 'default_cohort';
use constant DEFAULT_PROJECT             => 'default_project';
```

Which results in the following error:
```
Bareword "DEFAULT_PROJECT" not allowed while "strict subs" in use at /data/qpn/bin/mri/uploadNeuroDB/NeuroDB/objectBroker/ConfigOB.pm line 477.
```

Reply:
Thanks, it was probably introduced by a merge conflict not properly resolved 😄

Reply:
@cmadjar I will retest to make sure something else was not broken